localstack/services/lambda_/resource_providers/aws_lambda_function.py

```diff
@@ -189,7 +189,7 @@ def send(event, context, responseStatus, responseData, physicalResourceId=None,
     var parsedUrl = url.parse(event.ResponseURL);
     var options = {
         hostname: parsedUrl.hostname,
-        port: 443,
+        port: parsedUrl.port,
```

Comment:
Careful with that. Please at least add a comment explaining that this is a change that we made (and why), otherwise it's going to be hard to track any changes we've made to the official cfn-response code.